data/grc/Erbse_1961_Xenophon-Opuscula/stemma.gv

```diff
@@ -0,0 +1,11 @@
+digraph {
+archetyp[color="grey", label="ω"] 
+Vind [label="Vind."] 
+Vaticanus989 [label="Vat. 989"] 
+hyparchetypy [label="y", color="grey"] 
+groupd [label="d", color="grey"] 
+archetyp -> Vind 

```

Comment:
What does
![image](https://user-images.githubusercontent.com/1204247/121152982-f72aad80-c845-11eb-82fd-944b5fcda2e3.png)
mean ?

Reply:
If I remember correctly, it means that there might be many intermediary nodes; but I'll check next week.

Reply:
Yes, it means that there are many intermediary steps between the archetype and this codex. I don't know if it is worth encoding this information or which attributes to use for this. I'll open an issue so that we can merge this.
#69